### PR TITLE
Show when a search doesn't match any apps/metrics/pings (fixes #146)

### LINKED ISSUE #146: Improve filter
In the filter box when we search for something and it doesnt match with any item, just a blank page stares at us. There should a text telling search doesnt match with any application or martics.

### PR DESCRIPTION
**Pull request by mhmohona**
This is a fix for  #146. 

Added a message so when search input doesn't meet with application or matrics name it will show a text like this: 

filter

### CODE REVIEW

File: src/pages/AppDetail.svelte [hunk 1]
```diff
@@ -49,7 +50,11 @@
       <td><code>{app.app_id}</code></td>
     </tr>
   </table>
-
+  {#if app.prototype}
+    <AppAlert
+      message="The {params.app} application is a prototype. The metrics and pings listed below may contain inconsistencies and testing strings."
+      bgColor="#808895" />
+  {/if}
```
Comment (wlach) [inline src/pages/AppDetail.svelte:57]: This seems unrelated to this change?

File: src/pages/AppDetail.svelte [hunk 2]
```diff
@@ -67,6 +72,8 @@
         <a href={`/apps/${params.app}/metrics/${metric.name}`}>{metric.name}</a>
         <i>{metric.description}</i>
       </li>
+    {:else}
+      <p>Your search didn't match any metric.</p>
```
Comment (wlach) [inline src/pages/AppDetail.svelte:76]: There's a bit of overlap between this and #129. We should probably account for the possibility that there are no metrics associated with an application (however rare that possibility may be in practice).

File: src/pages/AppList.svelte [hunk 3]
```diff
@@ -36,7 +36,12 @@
         {#if app.deprecated}
           <Pill message="Deprecated" bgColor="#4a5568" />
         {/if}
+        {#if app.prototype}
+          <Pill message="Prototype" bgColor="#808895" />
+        {/if}
```
Comment (wlach) [inline src/pages/AppList.svelte:41]: This also looks unrelated.

**Review by wlach — CHANGES_REQUESTED** [review]
Can you give this pull request a more descriptive title? It would also be a good idea to specify which issue it fixes in the title (that way we can link pull requests with issues automatically)

File: src/pages/AppDetail.svelte (hunk 1 above)
Comment (mhmohona) [reply src/pages/AppDetail.svelte:57]: I did not add this. I only added the else part for this PR. 
I am confused why it has come into my edit. :sweat_smile:

Comment (mhmohona) [reply src/pages/AppDetail.svelte:57]: The formate changed after I ran `npm run format`. 
Otherwise, I was getting error on CircleCI for not running Prettier.

File: src/pages/AppDetail.svelte (hunk 2 above)
Comment (mhmohona) [reply src/pages/AppDetail.svelte:76]: Yes, you are right, it is showing the error message when there is no metrics associated. 
image

Should I wait till #129 gets marged and after that work on this PR @wlach?

Comment (wlach) [reply src/pages/AppDetail.svelte:76]: Yes that makes sense! I expect it will be merged soon, the code looks good just waiting on @DawoodShahat to fix a merge conflict.

Comment (wlach) [reply src/pages/AppDetail.svelte:76]: #129 has been merged

**Review by wlach — APPROVED** [review]
Thanks!
